### ISSUE TITLE
Improve visibilty of open StackBlitz/github buttons

### ISSUE THREAD

**Issue by nuke-ellington** [siemens/ix#347]
### What happened?

Apparently the two buttons are hard to detect if one does not konw about them.
We should try to improve their visibilty.

**Comment by sn0wcat** [comment]
Yes! could we also please show them also on the preview page, not just on the code pages?

![image](https://user-images.githubusercontent.com/1426617/216561058-bdcb2b7e-043a-4ca3-8c7b-363f919bd71f.png)

**Comment by nuke-ellington** [comment]
> Yes! could we also please show them also on the preview page, not just on the code pages?
> 
> ![image](https://user-images.githubusercontent.com/1426617/216561058-bdcb2b7e-043a-4ca3-8c7b-363f919bd71f.png)

The links are tied to the various platforms therefore they only make sense in the context of a selected technology. We could think about linking to JavaScript per default.

**Comment by sn0wcat** [comment]
That sounds great - you could also have a dropdown button with icons where the user can select it, but defaulting to javascript will probably suffice.

**Comment by danielleroux** [comment]
[IX-837]

**Comment by matthiashader** [comment]
[IX-837]

**Comment by danielleroux** [comment]
Fixed with latest documentation redesign
